src/main/java/com/entrevistador/orquestador/dominio/model/enums/EstadoProcesoEnum.java

```diff
@@ -1,4 +1,26 @@
 package com.entrevistador.orquestador.dominio.model.enums;
 
+import lombok.Getter;
+
+@Getter
 public enum EstadoProcesoEnum {
+    AC("Analizando CV"),
+    AOE("Analizando oferta de empleo"),
+    CVA("Comprobando validez de archivo"),
+    AE("Analizando empresas"),
+
+    GF("Generando feedback"),
+    GFG("Generando feedback general"),
+
+    GP("Generando preguntas"),
+    OR("Obteniendo respuesta"),
+    AR("Analizando respuesta"),
+    AF("Acumular feedback");
+
+    private String descripcion;
+
+    private EstadoProcesoEnum(String descripcion){
```

Comment:
De acuerdo a Java, el constructor es implícitamente private. 
Una buena practica es evitar la redundancia.
```suggestion
    EstadoProcesoEnum(String descripcion) {
```

Reply:
No sabia que por defecto los constructores son privados, gracias. Ya lo cambio